src/components/modals/login/wallets/subWallet/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Chain, Wallet } from '@rainbow-me/rainbowkit'
+import { SubWalletConnector } from '@subwallet/wagmi-connector'
+
+export interface MyWalletOptions {
+  chains: Chain[]
+  shimDisconnect?: boolean
+}
+
+export const subWalletWallet = ({ chains, shimDisconnect }: MyWalletOptions): Wallet => ({
+  id: 'subwallet',
+  name: 'SubWallet',
+  iconUrl: '/subwallet.svg',
+  iconBackground: '#fff',
+  downloadUrls: {
+    browserExtension: 'https://subwallet.app/download.html'
+  },
+  createConnector: () => {
+    const connector = new SubWalletConnector({
+      chains: chains as any,
+      options: {
+        shimDisconnect
+      }
+    })
+
+    return {
+      connector: connector as any
+    }
+  }
+})
```

Comment:
pls format this file with prettier, along with the talisman/index.ts one

actually I'm curious, why is this file not formatted by prettier? isn't it in precommit? how did you add this file?